source/saml/tabs/rules.rst

```diff
@@ -0,0 +1,18 @@
+Rules for assigning authorisations
+----------------------------------
+
+
+It will be necessary to establish rules for assigning authorisations to your users (to give them a profile, for example).
+
+To do this, go to **Administration** > **Rules** > **Rules for assigning authorisations to a user**.
+
+There are no ‘mandatory’ rules, it’s up to you to create the rule(s) that you feel best suit your needs according to your available criteria.

```

Comment:
A hard limitation in the current plugin is that the rules can only be bound to the 'email' condition. We are planning to allow binding it to additional SamlClaims, currently it only allows the value communicated via the nameId property or emailaddress claim.